src/Components/ArticlePage.jsx

```diff
@@ -3,45 +3,48 @@ import Button from 'react-bootstrap/Button'
 import {useParams, Link } from 'react-router'
 import {useEffect, useState}from 'react'
 import axios from 'axios'
-import CommentCard from './CommentCard'
+import CommentList from './CommentList';
 
 function ArticlePage (){
     const {article_id} = useParams();
     const [article, setArticle] = useState([])
-    const [comments, setComments] = useState([])
+    const [votes, setVotes] = useState(0)
     useEffect(() => {
         const getArticleById = async () => {
             const { data } = await axios.get(`https://nc-news-lo7q.onrender.com/api/articles/${article_id}`)
             setArticle(data.article)
+            setVotes(data.article.votes)
+            console.log(data.article)
+          
         }
         getArticleById()
     }, [article_id])
 
-    useEffect(() => {
-        const getCommentsByArticleId= async () => {
-            const { data } = await axios.get(`https://nc-news-lo7q.onrender.com/api/articles/${article_id}/comments`)
-            setComments(data.comments)
-        }
-        getCommentsByArticleId()
-    }, [article_id])
-
+    const patchVotes = async (increment) => {
+        await axios.patch(`https://nc-news-lo7q.onrender.com/api/articles/${article_id}`, {
+           inc_votes: increment,
+         })
+         console.log(article)
+         setVotes((votes) => votes + increment)
+       }
+    
+    
 return  (
 <div className="card-container-article-page">
     <Link to={'/'}><Button variant="dark">Home Page</Button></Link>
     <Card style={{ width: "10rem" }} className="card-body-article-page" >
     <Card.Title className="article-page-header">{article.title}</Card.Title>
     <Card.Text>{article.author}</Card.Text>
     <img src={article.article_img_url} className="article-page-image"/>
+    <Card.Text>{article.body}</Card.Text>
     <Card.Text>{article.created_at}</Card.Text>
     <Card.Text>Topic: {article.topic}</Card.Text>
-    <Card.Text>Votes: {article.votes}</Card.Text>
+    <Card.Text>Votes: {votes}</Card.Text>
+    <Button onClick={() => patchVotes(1)}>Upvote</Button>
+    <Button onClick={() => patchVotes(-1)}>Downvote</Button>
```

Comment:
It's worth thinking about user experience here, by making a button `disabled` after it has been clicked you can prevent users from spamming the votes too much